project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.1.1
+sbt.version=1.3.2
```

Comment:
I switched this to use the same version used by your `sbt-spiewak-bintray` plugin.